Database/Books/BooksRepository.cs

```diff
@@ -14,56 +20,23 @@ public BooksRepository(LibraryDbContext dbContext)
-    //            //new() { Id = 7, Title = "The Great Gatsby", Author = "F. Scott Fitzgerald" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 8, Title = "To Kill a Mockingbird", Author = "Harper Lee"   , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 9, Title = "The Great Gatsby", Author = "F. Scott Fitzgerald" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 2, Title = "To Kill a Mockingbird", Author = "Harper Lee" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 3, Title = "The Great Gatsby", Author = "F. Scott Fitzgerald" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 4, Title = "To Kill a Mockingbird", Author = "Harper Lee" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 5, Title = "The Great Gatsby", Author = "F. Scott Fitzgerald"    , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 6, Title = "To Kill a Mockingbird", Author = "Harper Lee" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 7, Title = "The Great Gatsby", Author = "F. Scott Fitzgerald" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 8, Title = "To Kill a Mockingbird", Author = "Harper Lee" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 9, Title = "The Great Gatsby", Author = "F. Scott Fitzgerald" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 1, Title = "1984", Author = "George Orwell" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 2, Title = "To Kill a Mockingbird", Author = "Harper Lee" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 3, Title = "The Great Gatsby", Author = "F. Scott Fitzgerald" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 4, Title = "To Kill a Mockingbird", Author = "Harper Lee" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 5, Title = "The Great Gatsby", Author = "F. Scott Fitzgerald" , Stock = 10  , BorrowedCount = 0},
-    //            //new() { Id = 6, Title = "To Kill a Mockingbird", Author = "Harper Lee" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 7, Title = "The Great Gatsby", Author = "F. Scott Fitzgerald" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 8, Title = "To Kill a Mockingbird", Author = "Harper Lee" , Stock = 10, BorrowedCount = 0},
-    //            //new() { Id = 9, Title = "The Great Gatsby", Author = "F. Scott Fitzgerald" , Stock = 10, BorrowedCount = 0}
-    //        });
-    //}
-
-    public async Task<List<BookModel>> GetBooksFromDbAsync()
+    public async Task<List<BookModel>> GetBooksAsync()
     {
         var books = await _dbContext.Books.ToListAsync();
         return books;
     }
+
     public async Task<(bool success, string message)> DeleteBookAsync(int id)
     {
         var book = await _dbContext.Books.FindAsync(id);
```

Comment:
you first have to check if the book exists and then if it is already borrowed. Try to make these 2 checks with only 1 db call.